apps/docs/content/blog/v2.4.0.mdx

```diff
@@ -266,8 +266,20 @@ you need to update the import as follows:
 + import {cn} from "@nextui-org/theme"
 ```
 
-<Spacer y={4} />
+### Validation Behavior
+
+Since v2.4.0, we've changed the default validation behavior to `aria` wihch means no native validation is applied. If you wish to use the native validation, you can set `validationBehavior` to `native` to the input components or set it to the Provider as stated above.
+
+For those who use `validationBehavior="aria"`, `validate` will be no longer applied since it is only for native validation. Therefore, you need to switch to use `isInvalid` prop instead.
```

Comment:
The form does not prevent submission, but the [`validate` function will display error messages](http://localhost:6006/?path=/story/components-input--required).